client/student/app/lecture/comment.panel.html

```diff
@@ -0,0 +1,14 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <div class="panel-title">
+            <span>Comments And Questions</span>
+        </div>
+    </div>
+    <ul class="list-group">
+        <li class="list-group-item" 
+            data-ng-repeat="item in vm.list track by $index">
+            <b>{{item.text}}</b>
```

Comment:
As specified in the API, each comment will have a 'content' property, which is the same as 'text' here. You should change 'text' to 'content' here or we should change 'content' to 'text' in the API specification.